source/faq.rst

```diff
@@ -48,11 +60,47 @@ The ``egicli endpoint list`` command can help you to get that list:
    100IT               org.openstack.nova  https://cloud-egi.100percentit.com:5000/v3/
 
 The providers also generate dynamic information about their characteristics via
-the Argo Messaging System.
+the Argo Messaging System which is easily browsable from AppDB.
+
+
+How can I choose which site to use?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sites offer their resources to users through Virtual Organisations (VO). Before
+you can access a Federated Cloud site you need to join a Virtual Organisation
```

Comment:
The more logical flow would be to have users be part of a VO for some reason then getting the VO supported somewhere instead of asking to join a VO to access a specific site.

Reply:
I rephrased check it if makes sense now